tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 minversion = 1.6
-envlist = py35,py36,py27,pep8,pylint,cover
+envlist = py35,py36,py37,py27,pep8,pylint,cover
```

Comment:
Side note: since this list is only used when running `tox` without arguments, I'm somewhat included to just shorted it in `py3,py27,pep8,pylint,cover`.